packages/react-devtools/app.js

```diff
@@ -32,6 +32,11 @@ app.on('ready', function() {
     },
   });
 
+  // set dock icon for macos
+  if (process.platform === 'darwin') {
+    app.dock.setIcon(join(__dirname, 'icons/icon128.png'));
+  }
```

Comment:
Oh this is nice!

nit: Maybe use the [`os` package](https://nodejs.org/api/os.html) though?
```diff
diff --git a/packages/react-devtools/app.js b/packages/react-devtools/app.js
index e28e13e13..174766490 100644
--- a/packages/react-devtools/app.js
+++ b/packages/react-devtools/app.js
@@ -9,6 +9,7 @@
 
 const {app, BrowserWindow} = require('electron'); // Module to create native browser window.
 const {join} = require('path');
+const os = require('os');
 
 const argv = require('minimist')(process.argv.slice(2));
 const projectRoots = argv._;
@@ -33,7 +34,7 @@ app.on('ready', function() {
   });
 
   // set dock icon for macos
-  if (process.platform === 'darwin') {
+  if (os.platform() === 'darwin') {
     app.dock.setIcon(join(__dirname, 'icons/icon128.png'));
   }
 

```